portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1182,6 +1182,14 @@ definition {
 			locator1 = "FormFields#GRID_OPTIONS_OPTION_ROW_NAME");
 	}
 
+	macro notViewOptionRowName {
+		AssertElementNotPresent(
+			key_fieldName = "${fieldTitle}",
```

Comment:
@gleicelisbino, why did you decide to name the parameter as `fieldTitle` instead of `fieldName`?

Can we rename to `fieldName` to keep consistency?

Reply:
@natocesarrego  Okay!!! I will do this.